packages/blockchain-wallet-v4-frontend/src/scenes/CoinPage/components/AccountSelectBox/AccountSelectBox.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useMemo } from 'react'
+
+import { StyledSelectBox } from './AccountSelectBox.styles'
+import { AccountSelectBoxComponent } from './AccountSelectBox.types'
+
+const AccountSelectBox: AccountSelectBoxComponent = ({ items, label, onChange, value }) => {
+  const key: string = useMemo(() => {
+    return value ? JSON.stringify(value) : ''
+  }, [value])
+
+  return (
+    <StyledSelectBox
+      key={key}
+      meta={{}}
```

Comment:
should we make ability to pass some meta data or no needed?

Reply:
I added that empty object because the component require that prop but no key inside it.
Should I add something there @milan-bc ?